app/models/assignment.rb

```diff
@@ -32,17 +32,23 @@ def notify(receiver, of:, by:)
 
   class << self
     def between(start_date, end_date)
-      where('start_date <= ? AND end_date >= ?', end_date, start_date)
+      where arel_table[:start_date].lteq(end_date).and(arel_table[:end_date].gteq(start_date))
```

Comment:
```suggestion
      where start_date: nil..end_date, end_date: start_date..nil
```
This isn't requested per-se but if we're going for "get rid of ugly interpolations", I'd offer this before getting into arel

Reply:
I did this in order to be able to pass in other Arel nodes (e.g. by way of `.current`). With this change:

```
Roster.last.assignments.current
/home/matt/.rbenv/versions/3.0.5/lib/ruby/gems/3.0.0/gems/activerecord-7.0.6/lib/active_record/connection_adapters/abstract/quoting.rb:25:in `quote': can't quote Arel::Nodes::NamedFunction (TypeError)
```